examples/platforms/nrf528xx/src/platform-nrf5.h

```diff
@@ -197,4 +197,39 @@ otError nrf5SdErrorToOtError(uint32_t aSdError);
 void nrf5SdSocFlashProcess(uint32_t aEvtId);
 #endif // SOFTDEVICE_PRESENT
 
+#if OPENTHREAD_CONFIG_ENABLE_PLATFORM_FEM_CUSTOM_SOURCE
+/**
+ * Initialization of custom FEM.
+ *
+ */
+void nrf5RadioCustomFemInit(void);
```

Comment:
can this be put into platform-fem.h? And maybe we can make the prefix shorter as `nrf5Fem`. Then refactor the built-in fem code with the new APIs. Thoughts?

Reply:
Yes. We can update the code likes that.

Reply:
Updated